faqs/are-there-any-risks-associated-with-the-merge.md

```diff
@@ -3,9 +3,7 @@ title: Are there any risks currently associated with the merge?
 weight: 9.0
 attribution:
   -
-    name:
-    link:
-  -
-    name:
-    link:
----
\ No newline at end of file
+    name: Lamboshi
+    link: https://twitter.com/L_Nakaghini
+---
+There are always risks when making a large change to a protocol that is securing hundreds of billions of dollars of assets. The merge has been described as "Trying to replace the engine of an airplane while you are flying." Thankfully, the beacon chain has been running for several months now without issues. There are 4 unique client implementations, so if one has a bug users can always switch to another, and before launching on mainnet, the code used will be exhaustively checked, tested, and checked again.
```

Comment:
I'm a sucker for sources - anything you can link to for:
- hundreds of billions of dollars of assets
- "Trying to replace the engine of an airplane while you are flying."
would be 🙌